spring-cloud/spring-cloud-kubernetes-2/client-service/pom.xml

```
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>client-service</artifactId>
+	<name>client-service</name>
```

change tab to spaces
```suggestion
    <name>client-service</name>
```